### PR TITLE
feat: fd/install UI updates

### PR DESCRIPTION
**Pull request by fidiego**
### Description 

Applies a series of small UI changes and theme updates. 

### Tracking

Tracked Here: https://github.com/orgs/powertoolsdev/projects/16/views/12?pane=issue&itemId=69179244

- https://github.com/powertoolsdev/mono/issues/5167
- https://github.com/powertoolsdev/mono/issues/5168
- https://github.com/powertoolsdev/mono/issues/5169
- https://github.com/powertoolsdev/mono/issues/5170
- https://github.com/powertoolsdev/mono/issues/5171
- https://github.com/powertoolsdev/mono/issues/5176

### CODE REVIEW

File: theme.ts [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        "card-border-width": "1px",
+        "button-border-width": "0px",
+      },
+      borderRadius: {
+        "card-radius": "0.25rem",
+        "button-radius": "0.25rem",
+      },
+    },
+  },
+};
```
Comment (jordan-acosta) [inline theme.ts]: We don't want to check this file in.

So, this is a bit of an issue with our current development workflow for the installer. Ideally, we could .gitignore it, but in the forks we _do_ need to check the file in, so it's there to be loaded in the vercel deployment.

File: theme.ts.example [hunk 2]
```diff
@@ -16,6 +16,9 @@ export default {
         "card-background": colors.white,
         "accordion-header-color": colors.black,
         "accordion-header-background": colors.gray["100"],
+        "accordion-header-active-color": colors.black,
+        "accordion-header-active-background": colors.blue["100"],
+        "step-active-border-color": colors.green["500"],
```
Comment (jordan-acosta) [inline theme.ts.example]: very nice

File: components/Video.tsx [hunk 3]
```diff
@@ -7,7 +7,7 @@ interface IVideo {
 
 export const Video: FC<IVideo> = ({ className = "", src }) => {
   return (
-    <div className="relative pb-[56.25%] h-0 overflow-hidden max-w-full">
+    <div className="relative pb-[56.25%] h-0 overflow-hidden max-w-full rounded-md">
```
Comment (jordan-acosta) [inline components/Video.tsx]: We should add this border radius to the theme, alongside the card and button border radiuses.

**Review by jordan-acosta — CHANGES_REQUESTED** [review]

**Comment by fidiego** [comment]
@jordan-acosta this should be good to go

**Comment by fidiego** [comment]
@jordan-acosta this PR has been rebased and works with the upstream changes. 

Note: i modified the method for selecting darkMode so i could debug by toggling the setting in my browser dev tools.

**Review by jordan-acosta — CHANGES_REQUESTED** [review]
The colors on the stepper UI appear to be mis-matched. The step circles are green or orange based on status, but the lines are still using the primary color.

Screenshot 2024-07-08 at 11 57 29 AM


Also, so far, I don't think any of our vendors are using media queries for dark mode, and just use a single color scheme. Using media for dev makes sense, but we may want to leave in the selector setting most vendor will probably use.

```js
  darkMode: ["media", ["@media (prefers-color-scheme: dark)", "&:is(.dark *)"]],
  // Use this to statically set dark or light mode.
  // darkMode: "selector",
  forceDarkMode: false,
```

**Pull request merged**